packages/opentelemetry-cloud-trace-exporter/src/transform.ts

```diff
@@ -233,7 +290,12 @@ function valueToAttributeValue(
     case 'string':
       return {stringValue: stringToTruncatableString(value)};
     default:
-      return {};
+      if (stringifyArrayAttributes) {
+        return {stringValue: stringToTruncatableString(JSON.stringify(value))};
```

Comment:
It would be good to drop `undefined` explicitly, as `JSON.stringify()` ts type is incorrect https://github.com/microsoft/TypeScript/issues/18879

Reply:
Do you mean dropping undefined from the array values? Because the typing is such that the value itself cannot be undefined, `transformAttributeValues` also specifically checks for that ahead of calling `valueToAttributeValue`.

Values in the array could be though, they can also be nulls, should both of those be dropped from arrays?

Reply:
Ah sorry, I missed that you already filter the undefined out in the function  above 👍